src/utils.js

```diff
@@ -5,13 +5,29 @@ import fs from 'fs';
 import fg from 'fast-glob'; // flow-uncovered-line
 
 import {execCmd} from './execCmd';
+
+type NameToFiles = {[name: string]: string[], ...};
+
+/**
+ * @desc Read ./.geraldignore if it exists. Otherwise, read ./.gitignore.
+ * Split the file by newlines to serve as the list of files/directories to
+ * ignore for Gerald.
+ */
+const getGeraldIgnore = (): Array<string> => {
+    if (fs.existsSync('.geraldignore')) {
+        return fs.readFileSync('.geraldignore', 'utf-8').split('\n');
+    } else if (fs.existsSync('.gitignore')) {
+        return fs.readFileSync('.gitignore', 'utf-8').split('\n');
+    }
```

Comment:
I think you should split apart these if/elses and combine the result from both .geraldignore AND .gitignore. This way you don't need to re-ignore everything listed in .gitignore in your .geraldignore just because you have one.